### PR TITLE
USC developer provided updates

### PR DESCRIPTION
**Pull request by mark-cooper**

### CODE REVIEW

File: public/controllers/aeon_requests_controller.rb [hunk 1]
```diff
@@ -146,7 +146,7 @@ def ancestry_titles_for(record)
         raise RecordNotFound.new if not node["publish"] == true
 
         if node["node_type"] === "resource"
-          breadcrumb_title = (node["finding_aid_status"] === 'completed' and not node["finding_aid_title"].nil?) ? node["finding_aid_title"] : node["title"]
+          breadcrumb_title = (node["finding_aid_status"] === 'completed' and not node["title"].nil?) ? node["title"] : node["title"]
```
Comment (mark-cooper) [inline public/controllers/aeon_requests_controller.rb:149]: This is equivalent to: `breadcrumb_title = node["title"]` as `node["title"]` is going to be used regardless given the update. It looks like you want to prefer `node["title"]` even if "finding_aid_title" is defined and the status is completed?

File: public/controllers/aeon_requests_controller.rb [hunk 2]
```diff
@@ -157,7 +157,7 @@ def ancestry_titles_for(record)
     when "resource"
       # see public/app/controller/records_controller#resource
       resource = ResourceView.new(record)
-      breadcrumb_title = (resource.finding_aid_status === 'completed' and not resource.finding_aid_title.nil?) ? resource.finding_aid_title : resource.title
+      breadcrumb_title = (resource.finding_aid_status === 'completed' and not resource.title.nil?) ? resource.title : resource.title
```
Comment (mark-cooper) [inline public/controllers/aeon_requests_controller.rb:160]: Same as previous comment. This update is equivalent to `breadcrumb_title = node["title"]`.

File: public/controllers/aeon_requests_controller.rb [hunk 3]
```diff
@@ -175,14 +175,14 @@ def aeon_request_hash(record)
...
-    location    = locations_data_for(record).map{ |l| "#{l[:area]}, #{l[:sub_area]}" }.join("; ")
+    location    = locations_data_for(record).map{ |l| "#{l[:area]}" }.join("; ")
     item_volume = locations_data_for(record).map{ |l| "#{l[:sub_area]}" }.join("; ")
     callnum     = callnum_for(record)
 
     {
       title:       title,
       site:        site,
-      location:    location,
+      sub_location:    location,
```
Comment (mark-cooper) [inline public/controllers/aeon_requests_controller.rb:185]: I think this requires an update to the README under "How it maps". Might "location" sometimes be the correct choice for some institutions?

**Review by mark-cooper — COMMENTED** [review]
Need to clarify finding_aid_title situation and consider whether params should be more flexible or can generally use "sub_location".

File: public/controllers/aeon_requests_controller.rb (hunk 2 above)
Comment (mark-cooper) [reply public/controllers/aeon_requests_controller.rb:160]: Errr, equivalent to `breadcrumb_title = resource.title` =)

File: public/controllers/aeon_requests_controller.rb (hunk 1 above)
Comment (ballin22s) [reply public/controllers/aeon_requests_controller.rb:149]: you are correct. would you suggest that I change this on my end?

File: public/controllers/aeon_requests_controller.rb (hunk 2 above)
Comment (ballin22s) [reply public/controllers/aeon_requests_controller.rb:160]: yea just noticed.

File: public/controllers/aeon_requests_controller.rb (hunk 3 above)
Comment (ballin22s) [reply public/controllers/aeon_requests_controller.rb:185]: Yea, we will need to update the README file to reflect this change.

**Review by ballin22s — APPROVED** [review]

**Comment by mark-cooper** [comment]
https://github.com/archivesspace/aeon_requests/pull/2

**Pull request closed without merging**